wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -16,21 +16,23 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
+// TODO: [WPB-6647] when opening self profile ensure these alerts are shown and also don't block each other
```

Comment:
has it been checked? if not change to specific ticket

Reply:
created a specific ticket: [WPB-11951](https://wearezeta.atlassian.net/browse/WPB-11951)